infrastructure/app-api.tf

```diff
@@ -97,8 +97,8 @@ module "app_api" {
   }
 
   providers = {
-    azurerm         = azurerm
-    azurerm.tooling = azurerm.tooling
```

Comment:
This couldn't pass the validation with it in

│ Error: Cannot override provider configuration 
│ 
│   on app-api.tf line 101, in module "app_api":
│  101:     azurerm.tooling = azurerm.tooling   
│ 
│ The configuration of module.app_api has its own local configuration for azurerm.tooling, and so it cannot accept an overridden configuration provided by the root module.